addon/mixins/prop-types.js

```diff
@@ -33,41 +33,47 @@ const helpers = {
       return
     }
 
-    const propTypes = ctx.get('propTypes')
-
-    if (!propTypes) {
-      return
-    }
-
-    Object.keys(propTypes).forEach(name => {
-      const def = propTypes[name]
-
-      if (def === undefined) {
-        Ember.Logger.warn(`propType for ${name} is unknown`)
+    const propTypes = [].concat(ctx.get('propTypes'))
```

Comment:
I'm thinking we should actually name this variable `propTypesArray` and change `property` to `propTypes`. When I see this variable called `propTypes` I think it is of the format:

```js
{
  foo: PropTypes.string,
  bar: PropTypes.number
}
```

when in actuality if I'm understanding this change correctly is of the format:

```js
[
  {
    foo: PropTypes.string
  },
  {
    bar: PropTypes.number
  }
]
```